src/wp-content/themes/zippy-child/assets/js/enquiry/Enquiry.js

```diff
@@ -83,72 +82,75 @@ export class Enquiry {
   }
 
   initAddMutipleButton() {
-    const enquiryButton = $('#enquiry-button');
-    
-    if (enquiryButton.length) { 
-      enquiryButton.on('click', () => {
-        const quantities = $('.quantity input.qty');
+    const enquiryButton = $("#enquiry-button");
+
+    if (enquiryButton.length) {
+      enquiryButton.on("click", (e) => {
+        e.preventDefault();
+
+        const quantities = $(".quantity input.qty").val();
+        if (quantities > 0) {
+        }
```

Comment:
why leave this one in here ? Please remove this condition !

Reply:
removed